### ISSUE TITLE
Class "Behat\Mink\Mink" not found

### ISSUE THREAD

**Issue by SeriousJ55** [minkphp/Mink#859]
I installed everything properly. When I try to launch the "Usage Example," I get the error message "Class "Behat\Mink\Mink" not found."

I tried installing in many different ways and nothing worked. How to install?

**Comment by aik099** [comment]
Please specify the steps you've performed for Mink installing and the code you're using to run Mink.

Also, have you been able to perform the Composer Install step?

**Comment by SeriousJ55** [comment]
1. I installed with composer: `composer require behat/mink`
2. I ran the [usage example code](https://github.com/minkphp/Mink#usage-example)

**Comment by stof** [comment]
Have you included the Composer autoload.php file ? https://getcomposer.org/doc/01-basic-usage.md#autoloading

**Comment by SeriousJ55** [comment]
Yes, of course. Before the usage example code, I wrote the line `include_once("path_to_mink_folder/vendor/autoload.php");`
